test/test.js

```diff
@@ -3128,52 +3128,96 @@ describe('object checks', function() {
 describe('array checks', function() {
     describe('is.sorted', function() {
         it('should return true if given array is sorted', function() {
-            var arr = [1, 2, 3, 4, 5];
-            expect(is.sorted(arr)).to.be.true;
+            var arr1 = [1, 2, 3, 4, 5];
+            // default behaviour:
+            expect(is.sorted(arr1)).to.be.true; // increasing
+            // extended behaviour:
+            expect(is.sorted(arr1, '>=')).to.be.true; // increasing
+            expect(is.sorted(arr1, '>')).to.be.true; // strictly increasing
+            expect(is.sorted(arr1, '<=')).to.be.false; // decreasing
+            expect(is.sorted(arr1, '<')).to.be.false; // strictly decreasing
+
+            var arr2 = [5, 4, 4, 3, 1];
+            // default behaviour:
+            expect(is.sorted(arr2)).to.be.false;
+            // extended behaviour:
+            expect(is.sorted(arr2, '>=')).to.be.false;
+            expect(is.sorted(arr2, '>')).to.be.false;
+            expect(is.sorted(arr2, '<=')).to.be.true;
+            expect(is.sorted(arr2, '<')).to.be.false;
         });
         it('should return false if given array is not sorted', function() {
             var arr = [1, 2, 3, 5, 4];
+            // default behaviour:
             expect(is.sorted(arr)).to.be.false;
+            // extended behaviour:
+            expect(is.sorted(arr, '>=')).to.be.false;
+            expect(is.sorted(arr, '>')).to.be.false;
+            expect(is.sorted(arr, '<=')).to.be.false;
+            expect(is.sorted(arr, '<')).to.be.false;
         });
     });
     describe('is.not.sorted', function() {
         it('should return false if given array is sorted', function() {
-            var arr = [1, 2, 3, 4, 5];
-            expect(is.not.sorted(arr)).to.be.false;
+            var arr1 = [1, 2, 3, 4, 5];
+            // default behaviour:
+            expect(is.not.sorted(arr1)).to.be.false;
+            // extended behaviour:
+            expect(is.not.sorted(arr1, '>=')).to.be.false;
+            expect(is.not.sorted(arr1, '>')).to.be.false;
+            expect(is.not.sorted(arr1, '<=')).to.be.true;
+            expect(is.not.sorted(arr1, '<')).to.be.true;
+
+            var arr2 = [5, 4, 4, 3, 1];
+            // default behaviour:
+            expect(is.not.sorted(arr2)).to.be.true;
+            // extended behaviour:
+            expect(is.not.sorted(arr2, '>=')).to.be.true;
+            expect(is.not.sorted(arr2, '>')).to.be.true;
+            expect(is.not.sorted(arr2, '<=')).to.be.false;
+            expect(is.not.sorted(arr2, '<')).to.be.true;
```

Comment:
Duplicating tests for `not` and `any` seems like overkill. We should have tests that ensure `not` and `any` do their thing and then it's safe to assume the behavior applied to others. What we can do is instead ensure the `sorted` method exists on the `not` and `any` object which is something to check for.